lib/motsu/src/context/environment.rs

```diff
@@ -0,0 +1,83 @@
+//! Module with unit test EVM environment for Stylus contracts.
+
+/// Block Timestamp - Epoch timestamp: 1st January 2025 `00::00::00`.
+const BLOCK_TIMESTAMP: u64 = 1_735_689_600;
+/// Arbitrum's CHAID ID.
+const CHAIN_ID: u64 = 42161;
+
+/// Dummy contract address set for tests.
+const CONTRACT_ADDRESS: &[u8; 42] =
+    b"0xdCE82b5f92C98F27F116F70491a487EFFDb6a2a9";
+
+/// Externally Owned Account (EOA) code hash.
+const EOA_CODEHASH: &[u8; 66] =
+    b"0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470";
+
+/// Dummy msg sender set for tests.
+const MSG_SENDER: &[u8; 42] = b"0xDeaDbeefdEAdbeefdEadbEEFdeadbeEFdEaDbeeF";
+
+pub(crate) struct Environment {
```

Comment:
I remember, that the whole reason of `MockStorage` was to store not just storage bytes (`contract_storage`), but also fields that you've put into `Environment` struct

Reply:
Because these data are also "mocked" values for our test cases